script.js

```diff
@@ -0,0 +1,48 @@
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const buttonAdd = document.getElementById('add');
+let booksData = [];
+const check = localStorage.getItem('books');
+if (check != null) {
+  booksData = JSON.parse(check);
+}
+
+buttonAdd.addEventListener('click', () => {
+  const bookTitle = title.value;
+  const bookAuthor = author.value;
+  const bookData = [bookTitle, bookAuthor];
+  booksData.push(bookData);
+  const allData = JSON.stringify(booksData);
+  localStorage.setItem('books', allData);
+  // eslint-disable-next-line no-restricted-globals
+  location.reload();
```

Comment:
- Kindly make sure you are using a function to add new books. This is a project requirement so you should create a function and call it to add a new book. :+1: :smiley: :smiley: 
![image](https://user-images.githubusercontent.com/98363075/204405855-78a9a514-3a2e-4f17-bab4-e41d893347b5.png)

- One of the objectives of this project is to learn to use objects. Kindly make sure you are using an array of objects for this project instead of an array of arrays. :pray: :pray: 
![image](https://user-images.githubusercontent.com/98363075/204405893-516b52e2-a83f-4a53-9251-cdd95d8b3492.png)